packaging/deb/control/postinst

```diff
@@ -32,10 +32,14 @@ case "$1" in
   fi
 
   if [ ! -f $PROVISIONING_CFG_DIR ]; then
```

Comment:
Actually, this test is incorrect. It checks whether the provisioning dir is a file, which it probably never will be.

It should either be `-e` (exists) or `-d` (dir exists) instead of `-f` (file existst).